runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -112,12 +112,21 @@
 	@Override
 	protected void startUp() throws Exception
 	{
-		if (client.getGameState() == GameState.LOGGED_IN
-			&& config.amount() != -1
-			&& !config.taskName().isEmpty())
-		{
-			clientThread.invokeLater(() -> setTask(config.taskName(), config.amount()));
-		}
+			if (client.getGameState() == GameState.LOGGED_IN
+				&& config.amount() != -1
+				&& !config.taskName().isEmpty())
+			{
+				clientThread.invokeLater(() -> setTask(config.taskName(), config.amount()));
+			} 
+			if (config.streak() != -1)
```

Comment:
These two should probably also have the check for LOGGED_IN? Also you have an extra level of indentation.